docs/src/exact.md

```diff
@@ -485,19 +488,20 @@ problem in general. Calcium will sometimes fail even in elementary cases.
 Here is an example of two constant trigonometric identities where
 the first succeeds and the second fails:
 
-```julia
+```jldoctest; setup = :(C = CalciumField())
 julia> a = sqrt(C(2)) + 1;
 
 julia> cos(a) + cos(2*a) + cos(3*a) == sin(7*a//2)//(2*sin(a//2)) - C(1)//2
 true
 
 julia> sin(3*a) == 4 * sin(a) * sin(C(pi)//3 - a) * sin(C(pi)//3 + a)
 ERROR: Unable to perform operation (failed deciding truth of a predicate): isequal
+[...]
```

Comment:
Just out of curiosity, why add these `[...]` to errors?

Reply:
https://documenter.juliadocs.org/dev/man/doctests/#Exceptions told me 

> To indicate to readers that the output result is truncated and does not display the entire (or any of) the stacktrace you may write `[...]` at the line where checking should stop, i.e.

and I liked that